<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 64, 64);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>331</width>
     <height>481</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1.7px solid Yellow;
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="l_fup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border: none;</string>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_up">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
background-color: rgb(255, 255, 255);
border: 1.7px solid black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browse">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;나눔명조 ExtraBold&quot;;
border-color: rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>281</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;나눔명조 ExtraBold&quot;;

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="l_doid">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10.5pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
     </property>
     <property name="text">
      <string>Data Owner ID</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_viewer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
background-color: rgb(255, 255, 255);
border: 1.7px solid black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_doid">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
background-color: rgb(255, 255, 255);
border: 1.7px solid black;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cal_pse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;나눔명조 ExtraBold&quot;;
border: 1.7px solid rgb(45, 45, 185);
color: rgb(0, 0, 0);
background-color: rgb(191, 191, 191);</string>
     </property>
     <property name="text">
      <string>Calculate pseudonym</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10.5pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
     </property>
     <property name="text">
      <string>Pseudonym</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_pse">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
background-color: rgb(255, 255, 255);
border: 1.7px solid black;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="l_fc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(146, 208, 80);
border: none;
</string>
    </property>
    <property name="text">
     <string>File Content</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="l_title1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 13pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(146, 208, 80);</string>
   </property>
   <property name="text">
    <string>File Import</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;나눔명조 ExtraBold&quot;;

color: rgb(146, 208, 80);
background-color: rgb(64, 64, 64);</string>
   </property>
   <property name="text">
    <string>File Encryption</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_title2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 13pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(146, 208, 80);</string>
   </property>
   <property name="text">
    <string>File Encrypt</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>361</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1.7px solid Yellow;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QPushButton" name="btn_and">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;

border: 1.7px solid rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
    </property>
    <property name="text">
     <string>AND Operation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_or">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;

border: 1.7px solid rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
    </property>
    <property name="text">
     <string>OR Operation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Employee">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Employee</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Team_Leader">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Team Leader</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Project_Manager">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Project Manager</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_AssProject_Manager">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>AssProject Manager</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Manager">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Manager</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Admin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Non_Employee">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Non-Employee</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Unregistered">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Unregistered</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Registered">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Registered</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Student">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Student</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Trainee">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>Trainee</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_HR">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>HR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_up">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;

border: 1.7px solid rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
    </property>
    <property name="text">
     <string>UPLOAD</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_efc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;나눔명조 ExtraBold&quot;;
color: rgb(146, 208, 80);
border:none;</string>
    </property>
    <property name="text">
     <string>Encrypted File Content</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txt_efc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;
background-color: rgb(255, 255, 255);
border: 1.7px solid black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_encry">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;

border: 1.7px solid rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
    </property>
    <property name="text">
     <string>ENCRYPTION</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>811</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>frame</zorder>
  <zorder>l_title1</zorder>
  <zorder>title</zorder>
  <zorder>l_title2</zorder>
  <zorder>frame_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
